canonical/maas-ui
# docs: Add machine list bulk action errors to storybook MAASENG-2219

## Done

- Added machine list bulk action errors to storybook
- Added machine list single action errors to storybook

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Run `yarn storybook`
- Find "Single Machine Error" and "MultipleMachinesErrors"
- Make sure both render as expected

## Fixes

Fixes [MAASENG-2219](https://warthogs.atlassian.net/browse/MAASENG-2219)

## Screenshots

![image](https://github.com/canonical/maas-ui/assets/35104482/8a30b7e3-ba1b-4748-bc4a-79cc56dcea94)



[MAASENG-2219]: https://warthogs.atlassian.net/browse/MAASENG-2219?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
